avantis_trader_sdk/client.py

```diff
@@ -156,7 +156,7 @@ async def send_and_get_transaction_hash(self, signed_txn):
         Returns:
             The transaction hash.
         """
-        return await self.async_web3.eth.send_raw_transaction(signed_txn.rawTransaction)
+        return await self.async_web3.eth.send_raw_transaction(signed_txn.raw_transaction)
```

Comment:
this is no valid
`raw_transaction` does not exist

Reply:
Fixed..